From 8f336bdae8b96bdf23cb282fdefd04ab331a05b3 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:38:10 -0400
Subject: [alsa] pcm_dmaengine: Use the available wrapper to get physical width

Message-id: <140972823573-378-patchset-jkysela@redhat.com>
Patchwork-id: 91074
O-Subject: [RHEL 7.1 PATCH 377/442] ALSA: pcm_dmaengine: Use the available wrapper to get physical width
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

params_physical_width() is available via pcm_params.h

Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
Acked-by: Lars-Peter Clausen <lars@metafoo.de>
Acked-by: Vinod Koul <vinod.koul@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Mark Brown <broonie@linaro.org>

(picked from kernel: 732814c8ffedd33a6b54f149b07ddec13c49fce9)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/soc/soc-dmaengine-pcm.c b/sound/soc/soc-dmaengine-pcm.c
index 76cbb9e..d5611ec 100644
--- a/sound/soc/soc-dmaengine-pcm.c
+++ b/sound/soc/soc-dmaengine-pcm.c
@@ -65,7 +65,7 @@ int snd_hwparams_to_dma_slave_config(const struct snd_pcm_substream *substream,
  enum dma_slave_buswidth buswidth;
  int bits;
 
- bits = snd_pcm_format_physical_width(params_format(params));
+ bits = params_physical_width(params);
  if (bits < 8 || bits > 64)
   return -EINVAL;
  else if (bits == 8)
-- 
1.7.1